setup.py

```diff
@@ -64,7 +64,7 @@
         "eth-abi>=4.0.0",
         "eth-account>=0.8.0,<0.13",
         "eth-hash[pycryptodome]>=0.5.1",
-        "eth-typing>=3.0.0,<4.2.0",
+        "eth-typing>=3.0.0",
```

Comment:
I think we need to say `>=3.0.0,!=4.2.0` (or whatever that version was that was breaking)

Reply:
+1 for this, looks good otherwise

Reply:
Oh okay, thank you! Updated